internal/loader/configuration_setting_loader.go

```diff
@@ -354,6 +331,56 @@ func (csl *ConfigurationSettingLoader) CheckAndRefreshSentinels(
 	return sentinelChanged, refreshedETags, nil
 }
 
+func (csl *ConfigurationSettingLoader) CheckPageETags(ctx context.Context, provider *acpv1.AzureAppConfigurationProvider, eTags map[acpv1.Selector][]*azcore.ETag) (bool, error) {
```

Comment:
there are AzureAppConfigurationProvider in the ConfigurationSettingLoader, don't need to pass it as a parameter